<launch>
    <param name="acc_noise"       type="double" value="1e-2" />
    <param name="gyr_noise"       type="double" value="1e-4" />
    <param name="acc_bias_noise"  type="double" value="1e-6" />
    <param name="gyr_bias_noise"  type="double" value="1e-8" />
    <param name="p_I_GNSS_x"      type="double" value="0."/>
    <param name="p_I_GNSS_y"      type="double" value="0."/>
    <param name="p_I_GNSS_z"      type="double" value="0."/>
    <param name="publish_tf"      type="bool" value="true"/>
    <node name="nmea_topic_driver" pkg="nmea_navsat_driver" type="nmea_topic_driver" output="screen" />

    <node name="eskf_imu_gps_node" pkg="eskf_imu_gps" type="eskf_imu_gps_node" output="screen" />

    <node pkg="rviz" type="rviz" name="rviz" output="screen" 
      args="-d $(find eskf_imu_gps)/rviz/eskf_imu_gps.rviz" required="true">
    </node>
</launch>
